addons/sys_radio/XEH_postInit.sqf

```diff
@@ -1,5 +1,15 @@
 #include "script_component.hpp"
 
+if (isServer) then {
+    // Close the radio if it was opened in case of disconnection in order to prevend dead lock state
+    GVAR(playerDisconnected) = addMissionEventHandler ["HandleDisconnected", {
+        params ["_unit", "", "", ""];
```

Comment:
Remove empty params?

Reply:
Didn't we decide to keep empty parameters? At least this is what was applied to radio functions that use  ACRE2 events...

Reply:
Oh, right, I guess so! But it doesn't say which is which in here since no header.

Reply:
I will add those. However, I need more testing regarding this fix and how much impact has sending to  each player the radio list when it changes...